base/docs/helpdb/Base.jl

```diff
@@ -2265,11 +2265,11 @@ sortperm
 """
     mod2pi(x)
 
-Modulus after division by 2pi, returning in the range \[0,2pi).
+Modulus after division by `2pi`, returning in the range `\[0,2pi)`.
```

Comment:
I'd be fine with this, but I've encountered stiff resistance in the past to adding unrelated changes to a PR.